public/pages/CreateMonitor/components/HTTPInput/HTTPInput.js

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Almost all of the codes in this file is inherited from the original PR. I only set some `inputProps` for the 2 `FormikFieldNumber` component.

Reply:
I believe similar code is available inside Destination as well. I believe similar comment was there in previous PR. Feel free to create Issue to remove duplication.